### DISCUSSION TITLE
How to restore multiple corrupted beatmaps

### DISCUSSION THREAD

**Discussion by insecurestudios** [ppy/osu#26657] [Q&A]
Hello, I am having a similar issue as in #20021 where the metadata of my beatmaps was saved but the content was not. The answer there was to redownload the beatmaps. Is there a way to easily redownload every beatmap or will I just have to redownload each beatmap manually?

Edit: After redownloading a map, the sound and the map was restored, but the background and star rating were not. How do I fix this?

Edit 2: Alternatively, would there be a way to store the names of all currently downloaded beatmaps to put into a beatmap package or something similar for easy reinstallment?

[logs](https://github.com/ppy/osu/files/14002455/compressed-logs.zip)
![Screenshot 2024-01-21 12 42 57 PM](https://github.com/ppy/osu/assets/126529308/990a874f-d367-47b6-b345-429f334e8e01)

**Comment by peppy** [comment]
your `files` folder is missing. you'll need to restore it. if not there's really no simple way to recover currently.
